chainer/functions/connection/n_step_lstm.py

```diff
@@ -453,7 +453,7 @@ def n_step_lstm(
 
     """
 
-    xp = cuda.get_array_module(hx.data)
+    xp = cuda.get_array_module(hx, hx.data)
```

Comment:
I think it is good that `get_array_module` check the underlying arrays of `Variable`s. Could you make a PR?